packages/web-runtime/src/App.vue

```diff
@@ -321,11 +324,18 @@ export default {
 
       this.appNavigationVisible = false
     },
-    extractPageTitleFromRoute(route) {
-      const titleSegments = [this.configuration.theme.general.name]
 
-      if (route.meta.pageTitle) {
-        titleSegments.unshift(route.meta.pageTitle)
+    announceRouteChange(route) {
+      const pageTitle = this.extractPageTitleFromRoute(route, false)
+      const translated = this.$gettext(' Navigated to %{ pageTitle }')
```

Comment:
```suggestion
      const translated = this.$gettext('Navigated to %{ pageTitle }')
```
sorry, I meant this blank ;-)